### ISSUE TITLE
Deployment: What are the options?

### ISSUE THREAD

**Issue by khink** [dddomodossola/remi#275]
A question about deployment: I'd like to have a Remi app running on publicly accessible infrastructure, so users can try it out.

I've read the readme about "Remote access", which would require running it on a local computer and opening my home router. This is not what i had in mind.

I did search online a bit and found https://gitter.im/dddomodossola/remi/archives/2016/11/16, from which i deduce there are people running Remi on Heroku. I'm toying with that now, but i'm not experienced in Heroku and i'm getting some errors that i'm slowly working my way through, and learning.

But i'm wondering what is the easiest way of deploying a Remi app publicly, and what other people used. Any such knowledge would be welcome, and i'd gladly format it and submit a PR for the readme.

**Comment by dddomodossola** [comment]
Hello @khink ,

I did some tests in the past without success. But since then, lots of improvements were done in remi.
And so it could be used on heroku without problems.
In relation to this topic, one of the most important change is the ability to use the same tcp port for http and websocket requests.
I used remi successfully on repl.it . It's a really simple way to use remi online.
Just remember that remi is not suitable to build websites, don't confuse it with solutions as flask or django.

Regards
